AntistasiOfficial.Altis/CREATE/createSupplyBox.sqf

```diff
@@ -215,36 +246,27 @@ while {alive _crate AND (_marker in markerSupplyCrates)} do {
 			waitUntil {sleep 1; (!alive _crate) OR (([80,1,_crate,"BLUFORSpawn"] call distanceUnits) AND !([100, 1, _crate, "OPFORspawn"] call distanceUnits))};
 		};
 
-		// if unloading was finished,  escape to the outer loop
+		// if unloading was finished
 		if (!(_counter < _deploymentTime)) exitWith {
-			//You won this mission
+			// delete the map marker
+			deleteMarker _marker;
```

Comment:
Not sure if this is interfering with the spawn check. Has to be tested (minimal chance for failure tho)

Reply:
spawn check ? link ?

Reply:
I was wrong, even without the marker distancia3.sqf will work as it should. No problem here